source/connect/ssh-connection.txt

```diff
@@ -11,3 +11,156 @@ Proxy / SSH Tunnel Connection Tab
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Proxy / SSH Tunnel` tab.
+
+      You have the following connection options:
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50 
+      
+         * - Option
+           - Description 
+
+         * - SSH with Password 
+           - Connects to a deployment using a SSH Tunnel and has the option to secure 
+             the connection with a password. For information on SSH connections and 
+             the fields for this connection option, see :ref:`<ssh-fields>`.
+
+         * - SSH with Identity File 
+           - Connects to a deployment using a SSH Tunnel and uses an Identity File 
+             to secure the connection. For information on SSH connections and 
+             the fields for this connection option, see :ref:`<ssh-fields>`.
+
+         * - Socks5
+           - Connects to a deployment using a seperate proxy server. For more 
+             information on the fields for this connection option, see 
+             :ref:`<socks5>`. 
+
+      .. _ssh-fields:
+ 
+      SSH Connection
+      ~~~~~~~~~~~~~~
+
+      Use the following fields to connect to a deployment using 
+      :guilabel:`SSH with Password` or :guilabel:`SSH with Identity File`:
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50
+      
+         * - Field
+           - Description
+
+         * - SSH Hostname
+           - Bastion (jumpbox) hostname. This is the unique identifier 
+             (Fully Qualified Domain Name, or FQDN) for the computer to access. 
+
+         * - SSH Port
+           - Port used for the SSH connection. This defaults to 22, 
+             the standard port for SSH.
+
+         * - SSH Username
+           - User for which the profile to log into on the remote system. This 
+             will be the user for which you want to establish the SSH connection.
+
+         * - SSH Password 
+           - (Optional) Password used to secure the SSH connection. This is 
+             required if you are not using an identity file.
+
+         * - SSH Identity File
+           - File from which the identity (private key) for SSH public 
+             key authentication is read.
+
+             Unix or OS X: If using OpenSSH, identity files are found in the ~/.ssh 
+             directory. By default, the private key files have one of the following 
+             file names:
+
+             - ``id_dsa``
+             
+             - ``id_ecdsa``
+
+             - ``id_ed25519``
+ 
+             - ``id_rsa``
+
+             On Windows, the location of the identify files depends on your choice 
+             of SSH client, such as PuTTY. 
+
+         * - SSH Passphrase
+           - (Optional) If your private key is encrypted, provide the passphrase 
+             used to decrypt your private key, which is stored. 
```

Comment:
"which is stored." looks odd. Stored where? I would rewrite.

Reply:
Done.